uri-factory/index.js

```diff
@@ -3,9 +3,27 @@ const InvalidArgumentError = require('../invalid-arg-error')
 const stampit = require('stampit')
 
 module.exports = stampit()
+  .props({
+    supportedQueryParamNames: ['target', 'search', 'scope', 'field', 'format']
+  })
   .methods({
-    uriFor (params) {
+
+    normalizeQueryParams (rawParams, supportedOnly = false) {
+      const params = {}
+      for (const [key, value] of Object.entries(rawParams)) {
+        const lcKey = key.toLowerCase()
+        if (this.supportedQueryParamNames.includes(lcKey)) {
+          params[lcKey] = value
```

Comment:
Any chance of logging unsupported params here? (which I guess would be the `else` case)
This doesn't have to hold up rolling these changes if it's even practical

Reply:
I had almost exactly the same thoughts. I want to revisit unsupported params in general (see issue #4), but I don't want that to hold up this release, which fixes some things that are broken in production. At least for now these code changes make what we're doing more explicit and self-documenting.